### PR TITLE
Improve event types definition for openseadragon

### PR DESCRIPTION
**Pull request by ghost**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [OpenSeadragon API Docs](http://openseadragon.github.io/docs/OpenSeadragon.html)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/openseadragon/index.d.ts [hunk 1]
```diff
@@ -1282,157 +1317,464 @@ declare namespace OpenSeadragon {
...
+        preventDefault: boolean;
+    }
+
+    interface PressMouseTrackerEvent extends PointerMouseTrackerEvent {
+        buttons: number;
+    }
+
+    type PointerType = 'mouse' | 'touch' | 'pen';
+
+    enum EventPhase {
```
[the remark this answers is not in the document]
Comment (peterblazejewicz) [reply types/openseadragon/index.d.ts:1773]: this is part of my remark, these are not enums, these are IMO proper symbols, just not exported through standard library:
https://dom.spec.whatwg.org/#event
If you really need those, you can export as const.
But the question was, why do you want those at all, those are not directly related to this particular (why one would like to add additional domain types to existing library domain objects)

[the remark this answers is not in the document]
Comment (leclerc91) [reply types/openseadragon/index.d.ts]: Ok, I'm going to omit that enum as its semantics exist in the `Event` type.
Now the comment was clear, thank you.

File: types/openseadragon/index.d.ts [hunk 2]
```diff
@@ -1770,12 +1770,7 @@ declare namespace OpenSeadragon {
 
     type PointerType = 'mouse' | 'touch' | 'pen';
 
-    enum EventPhase {
-        NONE = 0,
-        CAPTURING_PHASE = 1,
-        AT_TARGET = 2,
-        BUBBLING_PHASE = 3,
-    }
+    type EventPhase = 0 | 1 | 2 | 3;
```
Comment (peterblazejewicz) [inline types/openseadragon/index.d.ts:1773]: Sorry to be a PITA with such niptick :)

**Comment by leclerc91** [comment]
Any other suggestions/comments/remarks? :)

**Comment by peterblazejewicz** [comment]
For me the change looks ok, but I'm not the owner of the implementation here (say to review changes for  interfaces splits), give it a chance and ping me if still waiting, thx!

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@leclerc91 thx!

**Comment by leclerc91** [comment]
Ready to merge

**Pull request merged**